platform/brains2/cmake/build_app.cmake

```diff
@@ -15,6 +15,7 @@ include(${COMMON_CMAKE}/modules.cmake)
 include(${COMMON_CMAKE}/lib.cmake)
 
 if(WITH_USB)
+  include(${COMMON_CMAKE}/add_git_defines.cmake)
```

Comment:
For now, the git information is only relevant if we're also building with USB support.